pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: layout.pug
+navigationTitle: Management cluster application requirements
+title: Management cluster application requirements
+menuWeight: 40
+excerpt: Management cluster application minimum resources and storage requirements
+beta: false
+enterprise: true
+---
+
+This section details requirements for management cluster-specific applications. For the list of all available platform applications, see [Platform Application Configuration Requirements](../../../projects/applications/platform-applications/application-requirements/).
```

Comment:
The link for `Platform Application Configuration Requirements` appears to be invalid. Appears it should be `../../projects/applications/platform-applications/application-requirements/`.

Reply:
actually this should be pointing to workspace platform applications
```suggestion
This section details requirements for management cluster-specific applications. For the list of all available platform applications, see [Platform Application Configuration Requirements](../../workspaces/applications/platform-applications/platform-application-requirements/).
```